doc/subsystems/networking/qemu_setup.rst

```diff
@@ -160,6 +170,22 @@ To enable IPv4 forwarding the following command should be run as root:
 
    sysctl -w net.ipv4.ip_forward=1
 
+Some applications may also require a DNS server. A number of in-tree samples
```

Comment:
I think you mean:

    A number of Zephyr-provided samples ...

Reply:
done